src/Serilog.Sinks.Datadog.Logs/Serilog.Sinks.Datadog.Logs.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard1.3;net45;net461;net472;net5.0</TargetFrameworks>
     <AssemblyName>Serilog.Sinks.Datadog.Logs</AssemblyName>
     <PackageId>Serilog.Sinks.Datadog.Logs</PackageId>
-    <PackageVersion>0.4.0</PackageVersion>
+    <PackageVersion>0.4.1-beta1</PackageVersion>
```

Comment:
note for the future: when we actually release this, let's make this a minor version bump (so `0.5.0`), since this a significant change in behavior.